scripts/check-ts.js

```diff
@@ -59,9 +59,38 @@ if (filterPackage) {
   }
 }
 
+let totalTsFiles = 0
+let totalJsFiles = 0
+
 packagesWithTs.forEach(project => {
+  const tsFiles = glob.sync(
+    `./packages/${project.split(/.*packages[/\\]/)[1]}/src/**/*.ts`,
```

Comment:
This only works when running from root repo directory, i.e. `node ./scripts/check-ts.js`. If CWD is `gatsby/scripts` then it fails, i.e. `node ./check-ts.js`

But I guess this is not critical.